### ISSUE TITLE
[QUESTION] Forcing node containers into 'dev' mode

### ISSUE THREAD

**Issue by alex-stout** [Azure/azure-functions-docker#897]
Is there a way to force the containers into `dev` mode to pickup on local.settings.json & dev port allocations?

**Comment by vijaykumar911** [comment]
Hi @alex-stout thanks for information will update you soon.

**Comment by vijaykumar911** [comment]
YES  you can force the containers into development mode to pick up on local.settings.json and development port allocations.
If you are using Docker to manage your containers, you can specify environment variables and port mappings in the docker-compose.yml file. You can override the values in local.settings.json with environment variables defined in the Compose file allowing you to customize the container behavior in development mode.

**Comment by alex-stout** [comment]
Thank you for the response. Just to clarify, I am interested what settings are needed to have the Docker function instance pick up the `local.settings.json` and dev port allocation. 

From my testing, simply having a the Dockerfile does not work.

**Comment by vivekjilla** [comment]
Thanks for the question. This has been inactive since 2023-05-23 (over three years), so we're closing it. If you still need help getting a node container to honor local.settings.json and dev port allocation on a current supported image, please open a new issue with your setup and we'll follow up.
